ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -140,6 +140,19 @@ private[testtool] final class ParticipantTestContext private[participant] (
   def currentEnd(overrideLedgerId: String): Future[LedgerOffset] =
     services.transaction.getLedgerEnd(new GetLedgerEndRequest(overrideLedgerId)).map(_.getOffset)
 
+  /**
+    * Gets the absolute offset that is beyond the current ledger end.
+    *
+    * Note: offsets are opaque byte strings, but they are lexicographically sortable.
+    * Prepending the current absolute ledger end with non-zero bytes creates an offset that
+    * is be beyond the current ledger end for the ledger API server.
+    * The offset might however not be valid for the underlying ledger.
+    * This method can therefore only be used for offsets that are only interpreted by the
+    * ledger API server and not sent to the ledger.
+    */
+  def futureOffset(): Future[LedgerOffset] =
```

Comment:
Can we rename this to something a but more deadly?

Perhaps "offsetWayPastLedgerEnd"?

Reply:
Renamed to `offsetBeyondLedgerEnd`